lib/metrics/context.js

```diff
@@ -118,14 +107,13 @@ module.exports = function (log, config) {
    *
    * @param data    target object
    * @param request request object
-   * @param event   event name
    */
-  function gather (data, request, event) {
+  function gather (data, request) {
     var metadata = request.payload && request.payload.metricsContext
     var token = request.auth && request.auth.credentials
     var doNotTrack = request.headers && request.headers.dnt === '1'
     var memcached = getMemcached()
-    var key = getKey(token, event)
+    var key = getKey(token)
```

Comment:
Should you guard against token missing uid or id? Similar to https://github.com/mozilla/fxa-auth-server/blob/2b85e026857fa8405fe4dbd090c42df61d11b643/lib/metrics/context.js#L63? Or possibly move `getKey` into resolve block?